build.gradle.kts

```diff
@@ -19,16 +18,20 @@ repositories {
 dependencies {
     val minestom = libs.minestom
 
-    compileOnly(minestom)
-    implementation(libs.zstd)
+    api(minestom)
```

Comment:
none of these should be api. Minestom is exposed in the api, however polar should not enforce a specific version of the Minestom dependency.